### PR TITLE
Cookie initialy baked by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "nautobot-app-v2.2.1",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-app-bgp-models.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-app-bgp-models",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-app-bgp-models",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "@glennmatthews @mzbroch",
        "full_name": "Network to Code, LLC",
        "email": "opensource@networktocode.com",
        "github_org": "nautobot",
        "app_name": "nautobot_bgp_models",
        "verbose_name": "Nautobot BGP Models App",
        "app_slug": "nautobot-bgp-models",
        "project_slug": "nautobot-app-bgp-models",
        "repo_url": "https://github.com/nautobot/nautobot-app-bgp-models",
        "base_url": "bgp",
        "min_nautobot_version": "2.0.0",
        "max_nautobot_version": "2.9999",
        "camel_name": "NautobotBGPModels",
        "project_short_description": "Nautobot BGP Models App",
        "model_class_name": "None",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/bgp-models/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/opt/ntc/drift-manager/outputs/.cookiecutters/cookiecutter-nautobot-app/nautobot-app",
        "_checkout": "nautobot-app-v2.2.1"
    },
    "base_branch": "develop",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ],
    "baked_commit_ref": "",
    "draft": true
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "json/bgp-models.json",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "nautobot-app-v2.2.1",
    "pull_request": null,
    "post_action": [
        "black"
    ],
    "disable_post_actions": false,
    "draft": true
}
```

### CODE REVIEW

File: development/Dockerfile [hunk 1]
```diff
@@ -19,20 +19,21 @@ FROM ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER}-py${PYTHON_VER}
 ARG NAUTOBOT_ROOT=/opt/nautobot
 
 ENV prometheus_multiproc_dir=/prom_cache
-ENV NAUTOBOT_ROOT ${NAUTOBOT_ROOT}
+ENV NAUTOBOT_ROOT=${NAUTOBOT_ROOT}
+ENV INVOKE_NAUTOBOT_BGP_MODELS_LOCAL=true
```
Comment (glennmatthews) [inline development/Dockerfile:23]: That seems weird to me - why are we baking an Invoke config into the Docker image?

File: nautobot_bgp_models/app-config-schema.json [hunk 2]
```diff
@@ -0,0 +1 @@
+true
```
Comment (glennmatthews) [inline nautobot_bgp_models/app-config-schema.json:1]: Should this be more intelligently generated based on the `default_statuses` default settings for this app?

**Review by glennmatthews — APPROVED** [review]

File: development/Dockerfile (hunk 1 above)
Comment (snaselj) [reply development/Dockerfile:23]: To be able to use `invoke` commands inside the container locally.

File: nautobot_bgp_models/app-config-schema.json (hunk 2 above)
Comment (snaselj) [reply nautobot_bgp_models/app-config-schema.json:1]: There will be another PR to create the app config schema. Can't automate that, generation is just a helper, and the file needs to be updated manually.

**Pull request merged**